src/db.py

```diff
@@ -117,19 +114,29 @@ def start_update(refresh_campus=False, recalculate_swipe=False):
                         Session.add_all(menu_items)
                         Session.commit()
 
-        Base.metadata.drop_all(bind=Engine, tables=[CollegetownEatery.__table__, CollegetownEateryHour.__table__])
-        Base.metadata.create_all(bind=Engine, tables=[CollegetownEatery.__table__, CollegetownEateryHour.__table__])
-        print("[{}] Fetching Collegetown eateries".format(datetime.now()))
-        yelp_query = collegetown_search()
-        collegetown_eateries = parse_collegetown_eateries(yelp_query)
-        Session.add_all(collegetown_eateries)
-        Session.commit()
+        if recalculate_swipe:
+            print("[{}] Updating swipe data".format(datetime.now()))
```

Comment:
Should we be using `get_today()` here?

Reply:
This is a pretty new addition, and a PR should be made to update some of the uses in date rather than always calling it I guess, but this PR is just for some of the more prevalent issues in the backend into release. This could be a new issue.